rustls/src/msgs/ffdhe_groups.rs

```diff
@@ -60,39 +60,244 @@ impl<'a> FfdheGroup<'a> {
 ///
 /// [RFC 7919 Appendix A.1]: https://datatracker.ietf.org/doc/html/rfc7919#appendix-A.1
 pub const FFDHE2048: FfdheGroup<'static> = FfdheGroup {
-    p: include_bytes!("ffdhe_groups/ffdhe2048-modulus.bin"),
+    p: &[
```

Comment:
On this commit, I checked we had working test coverage by introducing a one-bit error into all of these locally, and checking the validation caught it. It did.